priam/src/main/java/com/netflix/priam/defaultimpl/StandardTuner.java

```diff
@@ -213,7 +213,7 @@ private void writeCassandraSnitchProperties() {
         if (Strings.isNullOrEmpty(dcSuffix)) {
             properties.remove("dc_suffix");
         } else {
-            properties.put("dc_suffix", "_" + dcSuffix);
+            properties.put("dc_suffix", dcSuffix);
```

Comment:
I think the original suggestion to make it a dumb concatenation makes sense.  There's no harm in having a data center name like "us-eastqa", it's just less readable to people.